### ISSUE TITLE
Cannot use ;on-device gemini nano' functional-sample per the provided README instructions.

### ISSUE THREAD

**Issue by M-JULIANI** [GoogleChrome/chrome-extensions-samples#1367]
**Describe the bug**
[on-device gemini nano](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/ai.gemini-on-device) functional-sample is not reproducible as described in README.

I am able to build and load the extension as an unpacked extension, however once loaded it fails due to this relative import:
![Screenshot 2024-12-03 at 21 16 26](https://github.com/user-attachments/assets/03d89dab-289f-405e-ac0e-f6a75dffdf76)

**To Reproduce**
Follow the steps [here](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/ai.gemini-on-device):

> 1. Clone this repository.
> 2. Run npm install in the project directory.
> 3. Run npm run build in the project directory to build the extension.
> 4. Load the newly created dist directory in Chrome as an [unpacked extension (https://developer.chrome.com/docs/extensions/get-started/tutorial/hello-world#load-unpacked).
> 5. Click the extension icon.
> 6. Interact with the Prompt API in the sidebar.
> 7. If you use this sample as the foundation for your own extension, be sure to update the "trial_tokens" field [with your own origin trial token](https://developer.chrome.com/docs/web-platform/origin-trials#extensions) and to remove the "key" field in manifest.json.

**Expected behavior**
The expected behavior would be that once the previous instructions are followed, you are able to activate extension and use it / interact with it.

**Screenshots**
![Screenshot 2024-12-03 at 21 16 26](h
![Screenshot 2024-12-03 at 21 18 21](https://github.com/user-attachments/assets/ddc074d1-839d-478d-836e-c8b592169d48)
ttps://github.com/user-attachments/assets/03d89dab-289f-405e-ac0e-f6a75dffdf76)

**Comment by M-JULIANI** [comment]
by commenting out `DOMPurify` and `marked` libraries, was able to reproduce the demo. might be worth adding the steps required to get those libs to load correctly within index.js

**Comment by sebastianbenz** [comment]
Did you build the project and import the `dist` folder?

**Comment by M-JULIANI** [comment]
yeah copied over the entire ai.gemini-on-device folder, my bad we can close this issue.

**Comment by sebastianbenz** [comment]
You don't want to know how often that has happened to me (and I've built the sample...).
